doc/index.md

```diff
@@ -0,0 +1,39 @@
+# Building an EAP server for the cloud
+
+In order to provision an EAP server for the cloud, you have first to integrate the [EAP Maven plugin](https://github.com/jbossas/eap-maven-plugin/)
+into the pom.xml file of your project.
+
+The "package" goal of the plugin allows you to:
+
+* Provision an EAP server tailored to your needs.
+* Incorporate custom content (keystores, properties files, ...).
+* Execute CLI scripts.
+* Deploy your application.
```

Comment:
Deploy your application in the provisioned server

Reply:
Fixed.